robot/basestation/templates/Navbar.html

```diff
@@ -45,10 +45,25 @@
         <button type="button" id="send-serial-btn" class="btn btn-primary">Send serial cmd</button>
       </div> -->
 
-      <div class="col-md-7">
-        <div class="row" style="color:white">
-          <div class="col-md-3">BATTERY VOLTAGE: </div>
-          <div class="col-md-1" id="battery-voltage">-</div>
+  <div class="col-md-3">
+    <div class="row" style="color:white">
+      <div class="col-md-4">BATTERY VOLTAGE: </div>
+      <div class="col-md-2" id="battery-voltage">-</div>V
+    </div>
+  </div>
+
+  <div class="col-md-5">
+    <div class="row" style="color:white">
+      <div class="col-md-3">BATTERY TEMPS: </div>
+      <div class="col-md-5">
+        <div class="row">
+          T2:<div class="col-md-3" id="battery-temp-1">-</div>&#8451;
+          T1:<div class="col-md-3" id="battery-temp-2">-</div>&#8451;
+        </div>
+        <div class=row>
+          T3:<div class="col-md-3" id="battery-temp-3">-</div>&#8451;
```

Comment:
Also I think a nice to have would be to simply add a hover to explain what the battery temperature is for. (Like what does T1 correspond to, T2 and T3)

This can be done with `<div title="Text that is hovering"></div>`